.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.4.9","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk1.8.0_241.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/dengnyuar/Library/Application Support/JetBrains/IdeaIC2021.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```

Comment:
whats this file?

Reply:
Not sure actually

Reply:
https://scalameta.org/metals/blog/2020/11/06/sbt-BSP-support.html
This article outlines it. I think it is autogenerated

Reply:
I've removed the file